src/ansys/dpf/core/__init__.py

```diff
@@ -1,8 +1,12 @@
 import os
 import pkg_resources
 
-from ansys.dpf.core._version import __version__
+try:
+    import importlib.metadata as importlib_metadata
+except ImportError:  # Python < 3.10 (backport)
+    import importlib_metadata as importlib_metadata
 
+__version__ = importlib_metadata.version("ansys-dpf-core")
```

Comment:
Where does it get the version from exactly? @PProfizi

Reply:
@BClappe from the `pyproject.toml`, meaning we have only one source of truth, and it is in the package metadata file.

Reply:
This is part of the work to adhere to PyAnsys best practices.